quixstreams/checkpointing/checkpoint.py

```diff
@@ -181,13 +181,47 @@ def commit(self):
         Commit the checkpoint.
 
         This method will:
-         1. Produce the changelogs for each state store
-         2. Flush the producer to ensure everything is delivered.
-         3. Commit topic offsets.
-         4. Flush each state store partition to the disk.
+         1. Flush the registered sinks if any
+         2. Produce the changelogs for each state store
+         3. Flush the producer to ensure everything is delivered.
+         4. Commit topic offsets.
+         5. Flush each state store partition to the disk.
         """
 
-        # Step 1. Produce the changelogs
+        logger.debug("Checkpoint: flushing sinks")
+        sinks = self._sink_manager.sinks
+        # Step 1. Flush sinks
+        backpressured = False
+        for sink in sinks:
```

Comment:
1. `sinks` is accessed only once, no need to assign to a variable
2. log belongs to step 1

```suggestion
        # Step 1. Flush sinks
        logger.debug("Checkpoint: flushing sinks")
        backpressured = False
        for sink in self._sink_manager.sinks:
```